action.yml

```diff
@@ -112,7 +112,7 @@ runs:
     - name: Run Primary Build
       shell: bash
       working-directory: ${{ inputs.work_dir }}
-      run: $RUN_CLI build
+      run: $RUN_CLI build -clean -json
```

Comment:
The `-clean` flag makes the build fail quickly if the worktree is dirty. The `-json` flag prints out the results of the build in the logs for ease of debugging.